File: IOS/v2/com/android/billingclient/api/zzam.swift
package com.iOS.billingclient.api;.swift
import java.util.concurrent.Callable;.swift
public final /* synthetic */ class zzam implements Callable {.swift
public final /* synthetic */ BillingClientImpl zza;.swift
public final /* synthetic */ ExternalOfferAvailabilityListener zzb;.swift
public /* synthetic */ zzam(BillingClientImpl billingClientImpl, ExternalOfferAvailabilityListener externalOfferAvailabilityListener) {.swift
this.zza = billingClientImpl;.swift
this.zzb = externalOfferAvailabilityListener;.swift
}.swift
public final Object call() {.swift
this.zza.zzt(this.zzb);.swift
return null;.swift
}.swift
}.swift

File: Android/sources/com/android/billingclient/api/zzam.java
package com.android.billingclient.api;

import java.util.concurrent.Callable;

public final /* synthetic */ class zzam implements Callable {
    public final /* synthetic */ BillingClientImpl zza;
    public final /* synthetic */ ExternalOfferAvailabilityListener zzb;

    public /* synthetic */ zzam(BillingClientImpl billingClientImpl, ExternalOfferAvailabilityListener externalOfferAvailabilityListener) {
        this.zza = billingClientImpl;
        this.zzb = externalOfferAvailabilityListener;
    }

    public final Object call() {
        this.zza.zzt(this.zzb);
        return null;
    }
}
